website/docs/terms/cte.md

```diff
@@ -1,30 +1,186 @@
+Let’s break this query down a bit:
+
+1. In the first `import_orders` CTE, I am simply importing the `orders` table which holds the data I’m interested in creating the customer segment off of. Note that this first CTE starts with a `WITH` statement and no following CTEs begin with a `WITH` statement.
+2. The second `aggregate_orders` CTE utilizes the `import_orders` CTE to get a count of orders per user with a filter applied.
+3. The last `segment_users` CTE builds off of the `aggregate_orders` by selecting the `customer_id`, `count_orders`, and creating my `buyer_type` segment. Note that the final `segment_users` CTE does not have a comma after its closing parenthesis.
+4. The final `select * from segment_users` statement simply selects all results from the `segment_users` CTE. 
+
+My results from running this query look a little like this:
+
+| USER_ID | COUNT_ORDERS | BUYER_TYPE |
+|---|---|---|
+| 3 | 3 | super_buyer |
+| 64 | 1 | single_buyer |
+| 94 | 2 | regular_buyer |
+
+>If you are finding yourself using the same code for a certain CTE across multiple 
+queries or models, that’s probably a good sign that CTE should be its own [model](https://docs.getdbt.com/docs/building-a-dbt-project/building-models) or view.
+
+## CTE vs Subquery
+
+A <Term id="subquery" /> is a nested query that can oftentimes be used in place of a CTE. Subqueries have different syntax than CTEs, but often have similar use cases. We won’t go too deep into subqueries here, but we’ll highlight some of the main differences between CTEs and subqueries below.
+
+| CTE | Subquery |
+|---|---|
+| Typically more readable since CTEs can be used to give structure to your query | Typically less readable, especially if there are many nested queries |
+| Allows for recursiveness | Does not allow for recursiveness |
+| CTEs must have unique CTE_EXPRESSION_NAMES when used in a query | Subqueries don’t always have to be explicitly named |
+| CTEs cannot be used in a `WHERE` clause | Subqueries can be used in a `WHERE` clause |
+
+## Data warehouse support for CTEs
+
+CTEs are likely to be supported across most, if not all, [modern data warehouses](https://blog.getdbt.com/future-of-the-modern-data-stack/). Please use this table to see more information about using CTEs in your specific data warehouse.
+
+| Data Warehouse | Support CTEs? |
+|---|---|
+|[Snowflake](https://docs.snowflake.com/en/user-guide/queries-cte.html) | :white_check_mark: |
+|[AWS Redshift](https://docs.aws.amazon.com/redshift/latest/dg/r_WITH_clause.html) | :white_check_mark: |
+|[Google BigQuery](https://cloud.google.com/bigquery/docs/reference/standard-sql/query-syntax) | :white_check_mark: |
+|[Databricks](https://docs.databricks.com/sql/language-manual/sql-ref-syntax-qry-select-cte.html) | :white_check_mark: |
+|[Postgres](https://www.postgresqltutorial.com/postgresql-cte/) | :white_check_mark: |
+
+## TL;DR
```

Comment:
These sections were supposed to add some kind of conclusion to the content. It feels a little duplicative to add it at the top since it most mirrors the main definition at the top. I feel like we have two options here:

1. Change the header name to "Conclusion" so it's more clear.
2. Remove these sections altogether if we feel like they are too duplicative. 

What do you think?

Reply:
Oh, got it! I like the idea of a summary.

Reply:
Cool, I'll go ahead and changes.